bin/rsync.app

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+DEST_PREFIX=$1
+shift
+
+DEPLOY_DEST=${DEST_PREFIX}www/wordpress-hathitrust
+DEPLOY_SRC=/htapps/test.www/wordpress-hathitrust
+
+EXCLUDE=$(cat <<EOT
+  --exclude .github
+  --exclude .git
+  --exclude .gitignore
+  --exclude $DEPLOY_SRC/node_modules
```

Comment:
I *think* this should just be `--exclude node_modules`

Reply:
I wasn't sure on that one, thanks!